tests/integration/test_tickets.py

```diff
@@ -107,7 +108,7 @@ def test_tickets_PUT(client, auth_headers):
     assert response.json['assignedUserID'] == 3
     assert response.json['sender'] == 'user2 tester'
     assert response.json['assigned'] == 'user3 tester'
-    assert response.json['status'] == 'in progress'
+    assert response.json['status'] == TicketStatus.In_Progress
```

Comment:
for the payload on the test we should use a hard-coded string here, because that is what we would get from a request.
```suggestion
        'status': 'In Progress',
```